### PR TITLE
[Compute AG][GH#36810] Update JFrog Artifactory

### PR DESCRIPTION
**Pull request by divyabhushan-okta**
Reference: https://redlock.atlassian.net/browse/CWP-37561

### CODE REVIEW

File: compute/admin_guide/vulnerability_management/registry_scanning/configure_registry_scanning.adoc [hunk 1]
```diff
@@ -47,8 +47,6 @@ For the on-demand scan, you must pre-define the image registry scope in the regi
 
 Verify that the images in the repository are being scanned.
 
-NOTE: From 22.11 (Lagrange) release or later, you can add a maximum of 19,999 registry entries in **Defend > Vulnerabilities > Images > Registry settings**. 
-
 [.procedure]
 . Go to *Monitor > Vulnerabilities > Images > Registries*.
```
Comment (Pubs-MV) [inline compute/admin_guide/vulnerability_management/registry_scanning/configure_registry_scanning.adoc:51]: Have you added the limits for 19,999 registries ( Defend > Vuln > Images > Registry Settings)  and  up to 100,000 images (Monitor > Vulnerabilities > Images > Registries) on the Limits topic?

File: compute/admin_guide/vulnerability_management/registry_scanning/configure_registry_scanning.adoc [hunk 2]
```diff
@@ -64,7 +62,7 @@ The *Last scan time* of a registry is updated under *Registry details > General
 
 You can trigger an on-demand scan for individual images with the https://prisma.pan.dev/api/cloud/cwpp/registry#operation/post-registry-scan[Scan Registry API]. This feature allows you to scan the images immediately and not wait for the next periodic scan. You can trigger multiple on-demand image scans without interrupting the main registry scanning process. 
 However, every trigger is for a single image only.
-
++
```
Comment (Pubs-MV) [inline compute/admin_guide/vulnerability_management/registry_scanning/configure_registry_scanning.adoc]: do you need this + here? it seems like a paragraph and not a step or bullet where you have added the note.

File: compute/admin_guide/vulnerability_management/registry_scanning/configure_registry_scanning.adoc (hunk 1 above)
Comment (divyabhushan-okta) [reply compute/admin_guide/vulnerability_management/registry_scanning/configure_registry_scanning.adoc:51]: @Pubs-MV Yes in line no: 20

**Comment by Pubs-MV** [comment]
is this tested @divyabhushan

**Comment by divyabhushan-okta** [comment]
@Pubs-MV Yes.

**Review by Pubs-MV — APPROVED** [review]

**Pull request merged**